core/domain/exp_jobs_one_off_test.py

```diff
@@ -1084,7 +1086,9 @@ def test_for_textangular_validation_and_migration(self):
                 job_id))
 
         # Test that validation passes after migration.
-        self.assertEqual(actual_output, [])
+        # There should be no validation errors in the new (updated)
+        # explorations, but there are 16 validation errors in the old versions.
```

Comment:
versions --> explorations since the point here is that the exploration identity is completely different, right?

Reply:
Wait, actually, I'm confused by this comment. How many explorations are we dealing with? And aren't we looking only at the latest version of each exploration?

Should it be "there should be no validation errors in the newly-created exploration, but there are still 16 validation errors in the old exploration"?

/cc @bansalnitish